### ISSUE TITLE
Would be nice to have an option for Spinsys for SIMPSON

### ISSUE THREAD

**Issue by jkshenton** [stur86/magresview-2#17]
Expanding on issue raised by @AKM2022

**Comment by dch0ph** [comment]
Agreed. Being able to choose a set of atoms and write out the NMR parameters ready for a simulation is one of the key selling points. It fits naturally into a graphical programme since you you might well need to choose the spins interactively - indeed writing out whole unit cell would generally be a Very Bad Idea and should generate a warning. 

Use cases:
Selecting an individual C atom + all the H atoms (including within adjacent molecules) within a certain radius, including the dipolar couplings.

Selecting atom X sites and writing them out. With multiple sites, you often would *not* want to include couplings and would generally construct the spectrum as a sum of single spin simulations.
